### PR TITLE
Place images and Reports in a directory for each test

### PR DESCRIPTION
**Pull request by vsdevelope**
### Change description ###
Images captured - will be placed in a directory for each test. 
The name of the directory will be the same as the name of log file generated for the tet run

### CODE REVIEW

**Review by Mikekatende — DISMISSED** [review]

File: UI/Selenium/Hooks/Hooks.cs [hunk 1]
```diff
@@ -10,42 +12,51 @@
...
-using NLog.Web;
+using TestLibrary.Utilities;
 
 namespace SeleniumSpecFlow
 {
     [Binding]
-    public class Hooks //: ObjectFactory
+    public class Hooks 
     {
          public static RestClient restClient { get; private set; }
```
[the remark this answers is not in the document]
Comment (vsdevelope) [reply UI/Selenium/Hooks/Hooks.cs]: Deleted

File: UI/Selenium/appsettings.json [hunk 2]
```diff
@@ -46,7 +46,9 @@
       "ServiceUrl": "https://vh-Service-web-dev.hearings.reform.hmcts.net",
       "SoapApiUrl": "",
       "DefaultElementWait": "20",
-      "OneMinuteElementWait": "60"
+      "OneMinuteElementWait": "60",
+      "ReportLocation": "/TestResults/Reports",
```
[the remark this answers is not in the document]
Comment (vsdevelope) [reply UI/Selenium/appsettings.json:50]: Settings updated. Both environments have same paths in the format of "//"

File: UI/Selenium/appsettings.json [hunk 3]
```diff
@@ -56,7 +58,9 @@
       "ServiceUrl": "https://vh-service-web-aat.hearings.reform.hmcts.net",
       "SoapApiUrl": "",
       "DefaultElementWait": "20",
-      "OneMinuteElementWait": "90"
+      "OneMinuteElementWait": "90",
+      "ReportLocation": "//TestResults//Reports",
```
[the remark this answers is not in the document]
Comment (vsdevelope) [reply UI/Selenium/appsettings.json]: fixed, same as above comment

File: UI/Selenium/appsettings.json [hunk 4]
```diff
@@ -46,7 +46,9 @@
       "ServiceUrl": "https://vh-Service-web-dev.hearings.reform.hmcts.net",
       "SoapApiUrl": "",
       "DefaultElementWait": "20",
-      "OneMinuteElementWait": "60"
```
[the remark this answers is not in the document]
Comment (vsdevelope) [reply UI/Selenium/appsettings.json:49]: log file name updated (removed "log." from filename)

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Review by Mikekatende — APPROVED** [review]

**Pull request merged**